www/conf/conf.d/ssl.conf.var

```diff
@@ -1,8 +1,9 @@
-# According to https://ssl-config.mozilla.org/#server=nginx&version=1.16.1&config=intermediate&openssl=1.1.1c&guideline=5.4
+# According to https://ssl-config.mozilla.org/#server=nginx&version=1.16.1&config=intermediate&openssl=1.1.1c&guideline=5.4 and added PQC hybrids support
```

Comment:
I know this is just a comment pointing to a reference, but it should probably be updated to OpenSSL 3.2 at some point.